src/Workers/Indexers/Runes.ts

```diff
@@ -11,70 +10,55 @@ export const runesIndexer: IndexHandler = {
   name: "runes",
 
   async run(idx: Indexer, { height, log }) {
-    if (height < RUNES_BLOCK) {
-      return;
-    }
-    log(`[Runes indexer]`);
+    if (height < RUNES_BLOCK) return;
+    if (idx.blocks.length === 0) return;
+    console.log(`\n[Runes indexer]`);
     const network = getNetworkEnum();
-
     // order blocks
     const blocks = idx.blocks.sort((a, b) => a.height - b.height);
 
+    // check reorg for runes
+    const initialHeight = blocks[0].height;
+    const runesIndexHeight = await runes.getCurrentBlock();
+    if (runesIndexHeight && runesIndexHeight?.height > initialHeight) {
```

Comment:
```suggestion
    if (runesIndexHeight?.height > initialHeight) {
```

I think can omit the first check